apps/condo/domains/ticket/components/TicketForm/UpdateTicketForm.tsx

```diff
@@ -11,28 +11,39 @@ import { Ticket, TicketFile } from '@condo/domains/ticket/utils/clientSchema'
 import ActionBar from '@condo/domains/common/components/ActionBar'
 import { Loader } from '@condo/domains/common/components/Loader'
 import { REQUIRED_TICKET_FIELDS, TICKET_SOURCE_TYPES } from '@condo/domains/ticket/constants/common'
-import { FormResetButton } from '@condo/domains/common/components/FormResetButton'
 
 import { BaseTicketForm } from '../BaseTicketForm'
 import { ErrorsContainer } from '../BaseTicketForm/ErrorsContainer'
 
 export const ApplyChangesActionBar = ({ handleSave, isLoading }) => {
     const intl = useIntl()
     const ApplyChangesMessage = intl.formatMessage({ id: 'ApplyChanges' })
+    const CancelLabel = intl.formatMessage({ id: 'Cancel' })
+
+    const { push, query: { id } } = useRouter()
+    const onCancel = useCallback((resetFields) => {
+        resetFields()
+        push(`/ticket/${id}`)
+    }, [id, push])
 
     return (
         <Form.Item noStyle shouldUpdate>
             {
-                ({ getFieldsValue }) => {
+                ({ getFieldsValue, resetFields, isFieldsTouched }) => {
                     const { property, details, placeClassifier, categoryClassifier, deadline } = getFieldsValue(REQUIRED_TICKET_FIELDS)
                     const disabledCondition = !property || !details || !placeClassifier || !categoryClassifier || !deadline
-
+                    const isNotTouched = !isFieldsTouched()
```

Comment:
i think it's redundant

Reply:
I kept the logic. Previously, if there were no changes in the form, then the button was disabled and you could only exit by pressing back. Do you think it's worth giving the ability to redirect the user back by clicking the cancel button?